pages/deployment/gcp.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Deploy Memgraph on Google Cloud Platform virtual machine 
+description: Learn how to deploy your Memgraph instance on the Google Cloud platform. 
+---
+
+import { Card, Cards, Tabs, Tab } from 'nextra/components'
+
+# Deploy Memgraph in the Google Cloud Platform (GCP)
+
+This guide will show you how to deploy Memgraph on a Google virtual machine
+(VM). The guide will cover only the specific bits that are different from the
+general deployment guide you can find for native [linux](./linux.mdx) or
+[docker](./docker.mdx) deployments.
+
+This guide assumes you have a Google Cloud Platform account and are familiar
+with the Google console. If you are not, you can follow the [GCP Linux VM quick
+start guide
+documentation](https://cloud.google.com/compute/docs/create-linux-vm-instance).
+
+## Creating a new GCP VM
+
+The first step is to have a VM running on GCP, where you will deploy Memgraph.
+Since Memgraph works nicely with Linux distributions, it is recommended that
+Linux-based VMs be used.
+
+Below are some guidelines to consider if you are opening a new GCP VM for
+Memgraph. 
+
+### Picking the OS 
+
+During the VM creation, you need to pick the OS you wish to have on the VM. If
+you are going to use the Memgraph native installation, you need to pick a Linux
+distribution that Memgraph supports.
+
+Memgraph supports the following Linux distributions:
+
+- Ubuntu (20.04, 22.04)
+- Fedora (38, 39)
+- Debian (11, 12) 
+- Centos (7, 9) 
+- Amazon Linux (2)
+- RedHat (7, 9)
+- Rocky (9.3)
+
+This way, you will manage the native installation of Memgraph, which can bring
+some speed benefits. 
+
+If you use the Memgraph Docker image, you can pick any Linux distribution as
+long as you can run Docker on it.
+
+
+### Picking the VM type
+
+When creating the VM, you need to pick the VM type. The process highly depends
+on your actuall workload. Consider using a VM with more CPU and memory for heavy
+graph processing. 
+
+Memgraph, by default, stores all data in the RAM, if you are going to use the
+default storage mode (`IN_MEMORY_TRANSACTIONAL` or `IN_MEMORY_ANALYTICAL`), make
+a calculation [how much data you will
+need]((/fundamentals/storage-memory-usage#calculate-storage-memory-usage)).
+There is also an [easy-to-use
+calulator](https://memgraph.com/enterprise#ram-calculator) for approximating
+memory usage. The rule of thumb is to have double the memory of what your
+storage would take. You can go with less if you do not have a demanding
+workload. 
```

Comment:
Please refer to AWS comment for this section to fix this.

Reply:
Done